packages/now-cli/test/unit.js

```diff
@@ -810,50 +240,6 @@ test('getProjectName with a directory', t => {
   t.is(project, 'aa');
 });
 
-test('4xx error message with broken JSON', async t => {
-  const fn = async (req, res) => {
-    send(res, 403, `32puuuh2332`);
-  };
-
-  const url = await getURL(fn);
-  const res = await fetch(url);
-  const formatted = await responseErrorMessage(res, 'Not authenticated');
-
-  t.is(formatted, 'Not authenticated (403)');
-});
-
-test('4xx error message with proper message', async t => {
-  const fn = async (req, res) => {
-    send(res, 403, {
-      error: {
-        message: 'This is a test',
-      },
-    });
-  };
-
-  const url = await getURL(fn);
-  const res = await fetch(url);
-  const formatted = await responseErrorMessage(res);
-
-  t.is(formatted, 'This is a test (403)');
-});
-
-test('5xx error message with proper message', async t => {
-  const fn = async (req, res) => {
-    send(res, 500, {
-      error: {
-        message: 'This is a test',
-      },
-    });
-  };
-
-  const url = await getURL(fn);
-  const res = await fetch(url);
-  const formatted = await responseErrorMessage(res);
-
-  t.is(formatted, 'Response Error (500)');
```

Comment:
Why remove the response error tests here?

Reply:
The whole function is not used anywhere in the code